lib/Runtime/Library/JavascriptObject.cpp

```diff
@@ -1305,17 +1305,17 @@ namespace Js
         Var thisArg = JavascriptOperators::OP_GetThisNoFastPath(args[0], 0, scriptContext);
         RecyclableObject* obj = RecyclableObject::FromVar(thisArg);
 
-        Var propertyKey = args.Info.Count > 1 ? args[1] : obj->GetLibrary()->GetUndefined();
-        const PropertyRecord* propertyRecord;
-        JavascriptConversion::ToPropertyKey(propertyKey, scriptContext, &propertyRecord);
-
         Var getterFunc = args.Info.Count > 2 ? args[2] : obj->GetLibrary()->GetUndefined();
 
         if (!JavascriptConversion::IsCallable(getterFunc))
         {
             JavascriptError::ThrowTypeError(scriptContext, JSERR_FunctionArgument_NeedFunction, _u("Object.prototype.__defineGetter__"));
```

Comment:
I see. What's the issue we're fixing here? Does something observable happen in JavascriptConversion::ToPropertyKey?